rollup.config.mjs

```diff
@@ -0,0 +1,83 @@
+import path from 'path'
+import commonjs from '@rollup/plugin-commonjs'
+import {nodeResolve} from '@rollup/plugin-node-resolve'
+import babel from '@rollup/plugin-babel'
+import {terser} from 'rollup-plugin-terser'
+import transformConstants from './script/babel-transform-constants.mjs'
+import transformUndebug from './script/babel-transform-undebug.mjs'
+
+const configs = []
+
+if (process.env.BUILD === 'dist') {
+  configs.push({
+    input: ['./lib/index.js', './lib/stream.js'],
+    output: [
+      {
+        dir: 'dist',
+        format: 'esm',
+        preserveModules: true,
+        entryFileNames: '[name].mjs'
+      },
+      {
+        dir: 'dist',
+        format: 'cjs',
+        exports: 'named',
+        preserveModules: true,
+        entryFileNames: '[name].js'
+      }
+    ],
+    onwarn: (warning) => {
+      throw Error(warning.toString())
+    },
+    external: (id) => !id.startsWith('.') && !path.isAbsolute(id),
+    plugins: [
+      nodeResolve({browser: true}),
+      babel.babel({
+        babelHelpers: 'bundled',
+        plugins: ['babel-plugin-unassert', transformUndebug, transformConstants]
+      }),
+      commonjs()
+    ]
+  })
+}
+
+if (process.env.BUILD === 'size') {
+  configs.push({
+    input: './lib/index.js',
+    output: {
+      file: './micromark.min.js',
+      format: 'umd',
+      name: 'Micromark',
```

Comment:
```suggestion
      name: 'micromark',
```

I may have missed this before, but it’s not a class—the examples use `micromark`, so it’d be good to match that casing